.gitignore

```diff
@@ -0,0 +1,2 @@
+bower_components
+bower.json
```

Comment:
This file should be in your remote repo. It allows me to pull down the files required to run your app.